tools/builds/ebpf/compile.mk

```diff
@@ -0,0 +1,58 @@
+CC=clang
+CFLAGS=-O2 -g  -Wall -target bpf -I/usr/include/$(shell uname -m)-linux-gnu
+
+MACROS :=
+DEBUG ?=
+
+MESH_MODE ?= istio
```

Comment:
istio?

Reply:
It's a little bit stripped down version of merbridge's Makefile, so I didn't change anything except removing unnecessary targets

Reply:
Can you link to the original one in a comment?

Reply:
Can you link to the original one in a comment?